src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -36,11 +37,11 @@ internal const val HEALTH_KUSTO_QUERY =
 
 @Service("csmADX")
 @Suppress("TooManyFunctions")
-@ConditionalOnProperty(name = ["csm.platform.vendor"], havingValue = "azure", matchIfMissing = true)
```

Comment:
Do we have another vendor or do we assume `vendor=azure` when `use-internal-result-services=false`? Should we keep the conditional property also?

(Same on the second service)

Reply:
I'm pretty sure we won't really be going to go back to ADX + Eventhub on azure once we started using the internal result services, so we can assume for the ResultDataClient and EventBusClient that the only external service should be on Azure and nowhere else.

Reply:
Right now we can assume that ADXClient or EventHubClient will ve used only if `useinternalresultservices` is not true. If we have a case in future with ADX/EventHub Client and result data services, we will change the conditional at this moment. Same thing if we have a case with AWS/GCP/....

Reply:
:+1: